tensorflow_quantum/datasets/spin_system_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 The TensorFlow Quantum Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Test the spin system dataset"""
+import tensorflow as tf
+import numpy as np
+import cirq
+from tensorflow_quantum.datasets import spin_system
+
+
+class SpinSystemDataTest(tf.test.TestCase):
```

Comment:
Since it's possible that other SpinSystems may go into the `spin_system.py` module we might want to rename this class to indicate we are only testing the tfi_chain method and then make new classes for each of the new spin systems.